packages/grafana-toolkit/package.json

```diff
@@ -84,7 +84,7 @@
     "optimize-css-assets-webpack-plugin": "^5.0.3",
     "ora": "^4.0.3",
     "pixelmatch": "^5.1.0",
-    "pngjs": "^3.4.0",
+    "pngjs": "^6.0.0",
```

Comment:
Where do we use this lib? I couldn't find any reference in the code base. @dprokop